draft-bbs-signatures.md

```diff
@@ -809,19 +809,19 @@ A cryptographic hash function that takes as an arbitrary octet string input and
 
 - hash\_to\_field\_dst: Domain separation tag used in the hash\_to\_field operation
 
-- hashing\_elements\_to\_scalars: either hash_to_scalar using H (in this case H MUST be an XOF), or hash_to_field with the additional check and re-calculation of more elements until the desired number of non-zero field elements is returned (as described in [Hash to scalar](#hash-to-scalar)).
+- hashing\_elements\_to\_scalars: either hash_to_scalar using hash (in this case hash MUST be an xof), or hash_to_field with the additional check and re-calculation of more elements until the desired number of non-zero field elements is returned (as described in [Hash to scalar](#hash-to-scalar)).
```

Comment:
not sure what's the difference here between `hash_to_scalar` and `hash_to_field`?

Reply:
If I’m not mistaken hash_to_field does not check if the field element is 0. It also does not worry about DST's etc that are not needed for the intended use cases.